win32ss/user/ntuser/window.c

```diff
@@ -3352,98 +3352,79 @@ NtUserFindWindowEx(HWND hwndParent,
 }
 
 
-/*
- * @implemented
- */
+/* @implemented */
 PWND FASTCALL UserGetAncestor(PWND Wnd, UINT Type)
 {
-   PWND WndAncestor, Parent;
+    PWND Ret, Parent, pwndNode, pwndMessage;
```

Comment:
Keep "WndAncestor" (or "Ancestor") instead of "Ret".

Reply:
Please move the `PWND pwndNode` inside the `case GA_ROOTOWNER:` block.

Reply:
Fixed.